entities/dashboard/widget.json

```diff
@@ -0,0 +1,128 @@
+{
+    "type": "object",
+    "properties": {
+        "uid": {
+            "type": "string",
+            "description": "Unique identifier of the dashboardWidget object"
+        },
+        "app_code_name": {
+            "type": "string",
+            "description": "The code name of the app that owns the widget"
+        },
+        "display_name": {
+            "type": "object",
+            "description": "The widget display name in different languages"
```

Comment:
Change to "A JSON object representing the display name for different languages. If a language is not supported, the Widget will display 'en'.  For example {"en":"Monthly Payments"}